ui/src/app/base/hooks/node.test.tsx

```diff
@@ -144,4 +155,166 @@ describe("node hooks", () => {
       expect(result.current).toBe(true);
     });
   });
+
+  describe("useMachineActions", () => {
+    let state: RootState;
+    const HookWrapper = ({ action }: { action: MachineMenuAction }) => {
+      const actions = useMachineActions("abc123", [action]);
+      return (
+        <>
+          {actions.map((action, i) => (
+            <button {...action} key={i} />
+          ))}
```

Comment:
The use of `action` in two different scopes is a bit confusing.

Reply:
Thanks, fixed!